app/src/main/java/com/github/palFinderTeam/palfinder/utils/PrettyDate.kt

```diff
@@ -14,6 +14,8 @@ class PrettyDate(private val now: Calendar = Calendar.getInstance()) {
         const val ZERO = "just now"
         const val FUTURE = "in %s"
         const val PAST = "%s ago"
+        const val MIN_AGE = 13
```

Comment:
This could be added in a separate class because it is completely unrelated, maybe `Age`, or directly in the `UserProfile`

Reply:
Or `CriteriaFragment`